plugins/modules/proxysql_replication_hostgroups.py

```diff
@@ -278,17 +283,60 @@ def delete_repl_group(self, check_mode, result, cursor):
                              " mysql_replication_hostgroups, however" +
                              " check_mode is enabled.")
 
+    def update_check_type(self, cursor):
+        try:
+            query_string = \
+                """UPDATE mysql_replication_hostgroups
+                    SET check_type = %s
+                    WHERE writer_hostgroup = %s
+                """
```

Comment:
```suggestion
            query_string = ("UPDATE mysql_replication_hostgroups "
                            "SET check_type = %s "
                            "WHERE writer_hostgroup = %s")
```
I suggest using this format. It looks more compact and efficient in terms of passing less whitespaces.